frontend/tests/components/opportunity/OpportunityDescription.test.tsx

```diff
@@ -1,12 +1,10 @@
 /* eslint-disable @typescript-eslint/unbound-method */
+/* eslint-disable testing-library/no-node-access */
```

Comment:
are we ok with this, or would we rather add test-id or label to the elements we're trying to target here? I was hoping that testing library would have something like "text content for header" or something, but it doesn't seem to.